gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-general.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+import { ApiEntity, PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+
+import homePage from "@pageobjects/HomePage/HomePage"
+import apiHome from "@pageobjects/Apis/ApiHome"
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+
+// This is purely for PageObject example and on hold for now until policy studio changes are completed.
+
+
+    describe('API General Page', () => {
+        beforeEach(() => {
+          cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+        });
+
+        const HomePage = new homePage()
+        const ApiHome = new apiHome()
+        const ApiDetails = new apiDetails()
+
+        it('Verify Home Page', () => {
+            cy.visit('/')
+            cy.wait(1000)
+            cy.contains('Home board').should('be.visible');
+        });
+        
+        it('Verify General Page Elements', () => {
+            cy.visit('/#!/environments/default/apis/');
+            cy.getByDataTestId('search').click().type("Test")
+            cy.get('input').type('{enter}')
+            cy.wait(1000)
```

Comment:
These `cy.wait()` should not be necessary, try to avoid them as much as possible.

Reply:
Ideally I would waitForElement or something as the page sometimes takes a second to re-organise the list when searching, but if I wait for the 'Test (1)' element that I'm trying to pick for this test for example, then there's a chance it could be near bottom of the list and pass that step before doing the first() bit... Any ideas?